usmqe/api/tendrlapi/common.py

```diff
@@ -131,33 +132,60 @@ def get_job_attribute(self, job_id, attribute="status", section=None):
         else:
             return response.json()[attribute]
 
+    def get_job_messages(self, job_id):
+        """ Get messages from job specified by job_id.
+
+        Name:       "get_job_messages",
+        Method:     "GET",
+        Pattern     "jobs/:job_id:/messages",
+
+        Args:
+            job_id:     id of job
+        """
+        pattern = "jobs/{}/messages".format(job_id)
+        response = requests.get(
+            pytest.config.getini("usm_api_url") + pattern,
+            auth=self._auth,)
+        self.print_req_info(response)
+        self.check_response(response)
+        return response.json()
+
     def wait_for_job_status(
             self,
             job_id,
-            max_count=42,
+            job_time=10000,
+            update_time=1000,
             status="finished",
             issue=None,
-            sleep_time=5):
+            sleep_time=10):
         """ Repeatedly check if status of job with provided id is in required state.
+        It is time bounded by job_time and each event is time bounded by update_time
 
         Args:
             job_id: id provided by api request
-            max_count: maximum of iterations
+            job_time: number of seconds in which the task must be in required state
+            update_time: number of seconds in which the event of the task must be done
             status: expected status of job that is checked
             issue: pytest issue message (usually github issue link)
             sleep_time: time in seconds between 2 job status function calls
         """
 
-        count = 0
+        start_time = datetime.datetime.now()
+        job_timeout = datetime.timedelta(0, job_time, 0)
+        update_timeout = datetime.timedelta(0, update_time, 0)
         current_status = ""
+        messages_count = 0
         while current_status not in (status, "finished", "failed") and\
-                count < max_count:
+            datetime.datetime.now() - start_time <= job_timeout and\
```

Comment:
you know that this is against what you said in the parameter description. Job should be done in X seconds, but if its done in exactly X seconds, it fails with `<` sign.